libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -279,8 +279,10 @@ public void track(AnalyticsTracker.Stat stat, Map<String, ?> properties) {
 
         if (propertiesToJSON.length() > 0) {
             mNosaraClient.track(EVENTS_PREFIX + eventName, propertiesToJSON, user, userType);
+            AppLog.i(AppLog.T.STATS, "\uD83D\uDD35 Tracked : " + eventName + ", Properties: " + propertiesToJSON.toString());
```

Comment:
This line needs to be less than 120 characters or else Travis will yell at you.  You can add `AppLog.T` to the import statements and change all the `AppLog.T.*` instances to `T.*` to remove a little verbosity.

Also, was the extra space in `Tracked :` intentional?  Should the `Properties:` text have the same format or should `Tracked :` be `Tracked:`?

Reply:
> This line needs to be less than 120 characters or else Travis will yell at you.

TIL.

>  You can add AppLog.T to the import statements and change all the AppLog.T.* instances to T.* to remove a little verbosity.

Would you believe I had it that way initially but removed it for consistency with the rest of the file. 😬  I'll add it back.

> was the extra space in Tracked : intentional?

Not intentional.  I didn't notice until you pointed it out. Will nuke.